File: lib/src/series/mode.dart
/// Mode determines whether a cash flow occurs at the start or end of the
/// compounding period demarcated by cash flow frequency. The options are:
///
/// * advance - cash flows due at the beginning of a compounding period
/// * arrear - cash flows due at the end of a compounding period
///
enum Mode {
  advance,
  arrear,
}

File: src/series/mode.ts
/**
 * Mode determines whether a cash flow occurs at the start or end of the
 * compounding period demarcated by cash flow frequency.
 */
export enum Mode {
  Advance = 'advance', // cash flows due at the beginning of a compounding period
  Arrear = 'arrear'    // cash flows due at the end of a compounding period
}
